packages/webdriverio/webdriverio.d.ts

```diff
@@ -19,7 +27,13 @@ type ElementAsync = {
 } & AsyncSelectors;
 
 // Element commands that should not be wrapper with promise
-type ElementStatic = Pick<WebdriverIO.Element, 'addCommand' | 'selector' | 'elementId'>
+type ElementStatic = Pick<WebdriverIO.Element,
+    'addCommand'
+    | 'selector'
+    | 'elementId'
+    | 'element-6066-11e4-a52e-4f735466cecf'
```

Comment:
could you explain how it would works with it `element-6066-11e4-a52e-4f735466cecf`?

Reply:
@CrispusDH see https://www.w3.org/TR/webdriver1/#elements

`element-6066-11e4-a52e-4f735466cecf` is constant, same as `ELEMENT` in JSONWP